### PR TITLE
Dexfinance etf

### PR DESCRIPTION
**Pull request by tuco-gordian**
https://defillama.com/protocol/dexfinance#information
Now, only Dexfinance, Dexfinance ETF, and Dexfinance Dexswap should remain. I have renamed the folders so that it logically matches what should be displayed currently.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/dexfinance-etf exports TVL: 
        
 
 
    bsc                       1.82 k
    
    total                    1.82 k

**Comment by llamatester** [comment]
The adapter at projects/dexfinance-etf exports TVL: 
        
 
 
    base                      82.13 k
    arbitrum                  5.05 k
    pulse                     3.74 k
    bsc                       1.82 k
    manta                     1.05 k
    optimism                  542
    avax                      520
    fantom                    237
    
    total                    95.08 k

File: projects/dexfinance-dexswap/index.js
Comment (g1nt0ki) [inline projects/dexfinance-dexswap/index.js:1]: please revert this change, it breaks our server, folder names are hardcoded

Comment (g1nt0ki) [reply projects/dexfinance-dexswap/index.js:1]: also, if some protocols is deprecated, we still show it on defillama as it still is live on chain/for historical data

**Comment by g1nt0ki** [comment]
I have refactored and merged it here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/10813/files
